Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM quay.io/hari_balagopal/move2kube-website-base:v0.3.5
```

Comment:
We can put it either in Move2Kube or Konveyor orgs in quay.

Reply:
sure, can you add it? don't have the perms